### DISCUSSION TITLE
Vercel Analytics vs Google Lighthouse

### DISCUSSION THREAD

**Discussion by tomasvorel2** [vercel/community#568] [Help] [answered]
Hi everyone, 

We're using Next.js for our main marketing site (around 900 pages) and hosting it on Vercel. One of my goals for the next Q is to improve our Lighthouse score (currently the score is pretty poor, currently perf on mobile around 25). I've recently enabled Vercel analytics for our project and I'm getting drastically different results (99 on mobile, which is a massive difference).

My questions:
1. Does Vercel's solution work with different metrics? It would explain the massive difference.
2. How relevant both of these are in today's landscape (which one is the more representative/reliable source of information)?

I understand that every site should follow best practices, but I've also read articles that downplay the whole Lighthouse score metric as not really important. Because my project is a marketing website, SEO and organic traffic are top priorities. Should I aim for 100/100/100/100 scores or is Lighthouse really not that important?

Thanks!

**Comment by samcx** [comment, marked as answer]
@tomasvorel2 I would use both to test the performance of my website. They are not the same → https://vercel.com/docs/concepts/analytics/web-vitals#real-experience-score — Lighthouse runs in a simulation, whereas Analytics grabs real data from users accessing your site.
